observability/micrometer/src/main/resources/org/wildfly/extension/micrometer/LocalDescriptions.properties

```diff
@@ -5,3 +5,5 @@ micrometer.remove=Operation Removes subsystem
 micrometer.exposed-subsystems=The names of the WildFly subsystems that exposes their metrics (or '*' to expose any subsystem metrics).
 micrometer.prefix=Prefix prepended to the name of the WildFly metrics exposed by the HTTP endpoints.
 micrometer.security-enabled=True if authentication is required to access the HTTP endpoint on the HTTP management interface.
+micrometer.endpoint=The address to where metrics will be published. The OTLP default is http://localhost:4318/v1/metrics, though WildFly provides no default, so this must be configured manually.
+micrometer.step=The step size (reporting frequency) to use in seconds. The default is 60 seconds.
```

Comment:
Remove "The default is 60 seconds." from the description. The default value is in the model, e.g.:

```
            "step" => {
                "type" => LONG,
                "description" => "The step size (reporting frequency) to use in seconds. The default is 60 seconds.",
                "expressions-allowed" => true,
                "required" => false,
                "nillable" => true,
                "default" => 60,
                "access-type" => "read-write",
                "storage" => "configuration",
                "restart-required" => "all-services"
            }
```

Reply:
Changed.